Frends.ManagementApi.Request/Frends.ManagementApi.Request/Frends.ManagementApi.Request.cs

```diff
@@ -0,0 +1,105 @@
+﻿#pragma warning disable SA1503 // Braces should not be omitted
+namespace Frends.ManagementApi.Request;
+
+using Frends.ManagementApi.Request.Definitions;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+/// <summary>
+/// Frends Management API Task.
+/// </summary>
+public static class ManagementApi
+{
+    /// <summary>
+    /// Task for reading data from Frends Management API.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.ManagementApi.Request).
+    /// </summary>
+    /// <param name="input">Input parameters.</param>
+    /// <param name="options">Option parameters.</param>
+    /// <param name="cancellationToken">Token generated by Frends to stop this Task.</param>
+    /// <returns>Object { bool Success, dynamic Data, dynamic ErrorMessage }.</returns>
+    public static async Task<Result> Request([PropertyTab] Input input, [PropertyTab] Options options, CancellationToken cancellationToken)
```

Comment:
The Task is not able to execute POST, PUT and PATCH operations since there is no body available for user to give. In addition, /api/v0.9/processes/batch-import API takes the input as multipart/form-data, which is also not handled in the Task. If there will only be one Task for Management API it should support all possible requests to Management API.